docker-compose.yml

```diff
@@ -0,0 +1,37 @@
+app:
+  restart: always
+  build: ./app/
+  expose:
+    - "8000"
+  env_file: .env
+  command: uwsgi --http :8000 --wsgi-file app/main.py
+  environment:
+    - MEMCACHED_HOST=memcached:11211
+  links:
+    - memcached
+
+nginx:
+  restart: always
+  build: ./conf/nginx/
+  ports:
+    - "80:80"
+  links:
+    - app
+
+memcached:
+  image: memcached
+  restart: always
+  expose:
+    - "11211"
+
+redis:
+  image: redis
+  expose:
+    - "6379"
+
+celery:
+  image: celery
+  links:
+    - redis
+  environment:
+    - CELERY_BROKER_URL=redis://redis:6379
```

Comment:
Looks like we're rolling with redis-backed celery and nginx+uwsgi. It'd be great to document the reasoning, whether docs in the repo, something sent to the mailing list, or a blog post :+1: